cmd/dcrdata/public/scss/_variables.scss

```diff
@@ -52,3 +52,7 @@ $card-bg-secondary: #f8f8f9;
 $card-bg-secondary-dark: #222326;
 
 $dark-link-hover-color: #94ffca;
+
+// links
+$link-decoration: none !default;
+$link-hover-decoration: underline !default;
```

Comment:
No `!default` needed for these I think.  We are changing the bootstrap defaults, and those definitions in their variables.scss needs `!default` for sass to allow this, but we don't need it here.